### ISSUE TITLE
improvement: angle tool.

### ISSUE THREAD

**Issue by RAFALCO** [opengisch/QField#2109]
it would be interesting to have an angle measuring tool, in addition to the lengths. 
with the measure tool, when measuring the first 2 segments, the angle could be displayed instead of the GPS coordinates. 
![Document1 (1)](https://user-images.githubusercontent.com/41300254/131321498-ce609a8e-b718-4c1a-9721-280ea2271075.jpg)

## Additional information:
angles could be in degrees or in grades.

**Comment by VxTedxV** [comment]
Not instead of, rather in addition. Coordinates stand useful !

**Comment by RAFALCO** [comment]
for me I think it is necessary to remove the coordinates when 2 segments are measured and replace them with the angle measurement.
For 1 segment, or more than 2, when the angle is no longer of interest, display the coordinates again.

**Comment by Kevin-Cowie** [comment]
Yes please!  I am using QField to record grid positions for archaeological surveys and I would find an angle measuring tool really useful for setting out grids.
